### PR TITLE
ES6 dual package requirements

### PR DESCRIPTION
**Pull request by jeremierobert-coveo**
Just do some modifications to fix nitty-gritty issues I found.

And add some padding to the user action no action message.

### CODE REVIEW

File: scripts/before.deploy.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 const { readFileSync, writeFileSync } = require('fs');
 const { resolve } = require('path');
 
-const BRANCH = process.env.TRAVIS_BRANCH;
-if (!BRANCH) {
-    throw new Error('TRAVIS_BRANCH is not define in the current environment');
+const TAG = process.env.TRAVIS_TAG;
+if (!TAG) {
+    throw new Error('TRAVIS_TAG is not define in the current environment');
```
Comment (louis-bompart) [inline scripts/before.deploy.js]: defined :P

File: src/components/UserActions/UserActions.scss [hunk 2]
```diff
@@ -44,6 +44,11 @@ $search-ui-dropdown-header-color: #1d4f76;
         margin: 0.5em 0;
     }
 
+    .coveo-no-actions {
```
Comment (louis-bompart) [inline src/components/UserActions/UserActions.scss]: Euh? Out of scope I'd say ?

Comment (jeremierobert-coveo) [reply src/components/UserActions/UserActions.scss]: Can we include this in the scope ?

Comment (louis-bompart) [reply src/components/UserActions/UserActions.scss]: We can, sure, but can you update your pr description accordingly etc

**Pull request merged**